src/pykiso/cli.py

```diff
@@ -124,14 +125,16 @@ def get_logging_options() -> LogOptions:
     "--test-configuration-file",
     required=True,
     type=click.Path(exists=True, dir_okay=False, readable=True),
+    multiple=True,
     help="path to the test configuration file (in YAML format)",
 )
 @click.option(
```

Comment:
I think it goes to what @Pog3k says above. I think there should be only one log path and a split per file done in the given directory